src/clj/rems/db/api_key.clj

```diff
@@ -1,5 +1,16 @@
 (ns rems.db.api-key
-  (:require [rems.db.core :as db]))
+  (:require [rems.db.core :as db]
+            [rems.json :as json]))
+
+(def +all-roles+ ["applicant" "decider" "handler" "logged-in" "owner" "past-reviewer" "reporter" "reviewer"])
```

Comment:
these could be keywords, right? we also have rems.roles/+roles+ which might be deprecated now?

Reply:
Thanks, changed them to keywords (as well as changing +all-roles+ to be a set instead of a vector), and changed +roles+ to +db-roles+ and added a comment there to distinguish it from +all-roles+ (the difference being that +db-roles+ only contains those roles that can be given to users in the database).